.github/workflows/selfservice-web.yml

```
@@ -0,0 +1,90 @@
+name: Build - SelfService - Web
+
+on:
+  push:
+    branches:
+      - "**"
+    paths:
+      - "Source/SelfService/Web/**"
+      - ".github/workflows/selfservice-web.yml"
+  pull_request:
+    types: [closed]
+
+env:
+  PRERELEASE_BRANCHES: "" # Comma separated list of prerelease branch names. 'alpha,rc, ...'
+  SOURCE_PATH: "Source/SelfService/Web"
+  DOCKER_FILE: "Source/SelfService/Web/Dockerfile"
+  DOCKER_IMAGE_TAG: "dolittle/studio/self-service-web"
```

Small tweaks, I did this yesterday (if you were curious)

```suggestion
DOCKER_IMAGE_TAG: "dolittle/self-service-web"
CONTAINER_REGISTRY: docker.io
```